tests/vars.sample.yaml

```diff
@@ -1,11 +1,11 @@
 # Path to install_yamls repo
-install_yamls_path: /CUSTOMIZE_THIS
+install_yamls_path:  /home/zuul-worker/src/github.com/openstack-k8s-operators/install_yamls/
```

Comment:
Again this is an environment-specific value that won't work outside CI. I'd rather keep the sample shouting "hey CUSTOMIZE_THIS" rather than provide CI-specific values. CI should probably create `vars.yaml` rather than try to use `vars.sample.yaml` directly?

Reply:
Cool. That CI-specific vars.yaml can perhaps even live on the CI side rather than here in the repo? These files - inventory, vars.yaml and secrets.yaml - should be the interface between the test suite and the execution environment. So rather than having vars for all available execution envs in this repo, each execution env should carry its own definition of those 3 files i think.

Reply:
then are you OK to carry a ci specific make target here? problem is the existing target hard codes these vars (and whats more they are -e so we cant override them they have highest precedence)

Reply:
I'm thinking can the CI execution just create `tests/vars.yaml` and `tests/secrets.yaml`? There maybe something i'm missing though.